Changes

```diff
@@ -1,6 +1,39 @@
 Release history of Zonemaster
 
 
+v2024.1 2024-07-01
+
+ [Release information]
+ - Translations have not been fully updated in this release. They will
+   be updated in an upcoming release.
+
+ [Breaking changes]
+ - Removes deprecated profile property in Zonemaster-Engine.
+ - Removes deprecated command line option in Zonemaster-CLI.
+ - Requires database update in Zonemaster-Backend.
```

Comment:
The database migration should be moved to the Release information heading. In the Backend release notes it's listed under Release information, and I think that's the correct place. The database is not a public interface.

Reply:
OK, fixat.